middle_end/flambda2/tests/ref_to_var/unboxing_cse.ml

```diff
@@ -0,0 +1,9 @@
+external ( < ) : int -> int -> bool = "%lessthan"
+
+external opaque : 'a -> 'a = "%opaque"
+
+let f x z =
+  let[@inline] test (_, y) = if y < z then opaque 0 else opaque 0 in
+  let _ = test (0, x) in
+  (* will work if you delete this line *)
```

Comment:
The formatting of ocamlformat makes it very unclear to which line this is referring, maybe clarify it in the comment ?